Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cc-traits"
-version = "0.7.3"
+version = "0.8.0"
```

Comment:
Please let me take care of that. You can add your changes under an `[Unreleased]` section in the `CHANGELOG.md` file if anything.